check_metapackage.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
```

Comment:
All I can tell from the publicly available resource is the [repository the original file is copied from is BSD](https://github.com/jsk-ros-pkg/jsk_travis/blob/1dc06eea19e9ac479c786066c92423eaf4c9ce04/package.xml#L5), and the name of its organization.

For BSD, looks like it's better to add license and copyrights header per each file ([referred to this](http://stackoverflow.com/a/3053725/577001)).

Therefore I added the license (BSD) and copyright header (under U-Tokyo).